README.md

```diff
@@ -14,19 +22,42 @@ Modules have been moved to [Ansible repository](https://github.com/ansible/ansib
 * [Virtual Machine Instance ReplicaSet facts](tests/playbooks/k8s_facts_vmirs.yml)
 * [All Virtual Machine Instance facts](tests/playbooks/k8s_facts.yml)
 
-## Local testing
+For the full list see [tests/playbooks](./tests/playbooks).
+
+## How to run the tests?
+
+1. Clone this repository to the machine, where you can run `oc login` to your cluster:
 
-1. Run the tests as follows:
+```shell
+$ git clone https://github.com/kubevirt/ansible-kubevirt-modules.git
+$ cd ./ansible-kubevirt-modules
+```
+
+2. (Optional) Configure a virtual environment to isolate dependencies:
+
+```shell
+$ python3 -m venv env
+$ source env/bin/activate
+```
+
+3. Install dependencies:
+
+```shell
+$ pip install ansible
```

Comment:
this will bring the release ansible version. 
worth mentioning how to bring a specfic commit from devel branch as well, becuase sometimes we are testing a modules merged into devel branch and they will be released on future.